content/docs/0.7.x/operate/install/index.md

```diff
@@ -219,288 +227,224 @@ Every Keptn release provides binaries for the Keptn CLI. These binaries are avai
 
-* If you are using a **LoadBalancer** to expose Keptn, get the EXTERNAL-IP of the `api-gateway-ngix` using the next command. Consequently, the Keptn API endpoint is: `http://<ENDPOINT_OF_API_GATEWAY>/api`
+### Option 4: Access Keptn via a Port-forward
+This option exposes Keptn on a *cluster-internal* IP .
 
+1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `--use-case=continuous-delivery`. Furthermore, if you are on OpenShift, please add `--platform=openshift`.
   ```console
-kubectl -n keptn get service api-gateway-nginx
+  keptn install (--use-case=continuous-delivery) (--platform=openshift)
   ```
 
+1. **Setup a Port-Forward:** Configure the port-forward to the using the command below.
   ```console
-NAME                TYPE        CLUSTER-IP    EXTERNAL-IP                  PORT(S)   AGE
-api-gateway-nginx   ClusterIP   10.107.0.20   <ENDPOINT_OF_API_GATEWAY>    80/TCP    44m
-  ```
-
-</p>
-</details>
-
-<details><summary>Keptn API endpoint when exposing via **NodePort**</summary>
-<p>
-
-* If you are using a **NodePort** to expose Keptn, please note that you either need to be on the same network as your Kubernetes VM (e.g., when using K3s or Minikube), or you have to set up certain firewall rules, e.g. for GKE (this is beyond this documentation). 
-
-    * Get the port of `api-gateway-nginx`:
-
-    ```console
-    API_PORT=$(kubectl get svc api-gateway-nginx -n keptn -o jsonpath='{.spec.ports[?(@.name=="http")].nodePort}')
+  kubectl -n keptn port-forward service/api-gateway-nginx 8080:80
     ```
 
-    * Get the internal and external IP of current Kubernetes node:
+1. **Get Keptn endpoint:** 
+  The Keptn API endpoint is: `http://localhost:8080/api`
 
+    Optional: Store Keptn API endpoint in an environment variable:
     ```console
-    EXTERNAL_NODE_IP=$(kubectl get nodes -o jsonpath='{ $.items[0].status.addresses[?(@.type=="ExternalIP")].address }')
-    INTERNAL_NODE_IP=$(kubectl get nodes -o jsonpath='{ $.items[0].status.addresses[?(@.type=="InternalIP")].address }')
+    KEPTN_API_ENDPOINT=http://localhost:8080/api
     ```
 
-    * Define Keptn API Endpoint (either via the internal or external IP; try both if unsure): `http://${INTERNAL_NODE_IP}:${API_PORT}/` or `http://${EXTERNAL_NODE_IP}:${API_PORT}/`
-
-
-</p>
-</details>
-
-<details><summary>Keptn API endpoint when exposing via **OpenShift Routes**</summary>
-<p>
-
-* If you are on **OpenShift 3.11**, get the HOST of the `api` route using the next command. Consequently, the Keptn API endpoint is: `http://<HOST>/api`
-
-  ```console
-  oc get route api -n keptn
-  ```
   
-  ```
-  NAME        HOST/PORT                       PATH      SERVICES            PORT      TERMINATION   WILDCARD
-  api         api-keptn.x.x.x.xip.io                    api-gateway-nginx   http      edge/None     None
-  ```
+## Authenticate Keptn CLI
```

Comment:
Please do a find `#5-authenticate-keptn-cli` and replace with `authenticate-keptn-cli` in the entire docu.